scripts/utils/git

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env bash
+
+# Import utils
+source ${0%/*}/utils/message
+
+# Install/link GIT hooks
+# Example: install_hooks "pre-commit pre-push"
+install_hooks() {
+  # Check if .git/hooks directory exists
+  if [ ! -d ".git/hooks" ]
+  then
+    warning "Couldn't install hooks as the directory ${NC}.git/hooks/${RED}"\
+              "doesn't exists"
+    return
```

Comment:
As far as I know there is by default always a `.git/hooks` directory, thus I decided not to add the needed extra command to create that directory if it doesn't exists. Do you think we should add this case?